### PR TITLE
Homework/1

### PR DESCRIPTION
**Pull request by Laimonukas**
Learning git. yay

### CODE REVIEW

File: Homework/1/CalcBMI.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                return;
+            }
+            Console.WriteLine($"{personName} {personSurname} is {personAge} years old, his weight is {personWeight} kg and his height is {personHeight} cm.\n");
+            Console.WriteLine($"BMI is {personWeight / Math.Pow(personHeight / 100, 2):F2}");
+            Console.WriteLine("Do you want to do another person? y/n");
+            if (Console.ReadLine().ToLower() == "y")
+            {
+                RequestAndReadInput();
+            }
+        }
```
Comment (Almantask) [inline Homework/1/CalcBMI.cs]: Split into smaller functions.

File: Homework/1/CalcBMI.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                RequestAndReadInput();
+                return;
+            }
+            Console.WriteLine($"{personName} {personSurname} is {personAge} years old, his weight is {personWeight} kg and his height is {personHeight} cm.\n");
+            Console.WriteLine($"BMI is {personWeight / Math.Pow(personHeight / 100, 2):F2}");
+            Console.WriteLine("Do you want to do another person? y/n");
+            if (Console.ReadLine().ToLower() == "y")
+            {
+                RequestAndReadInput();
+            }
```
Comment (Almantask) [inline Homework/1/CalcBMI.cs]: all code is a blob, hard to read.

**Review by Almantask — CHANGES_REQUESTED** [review]

File: Homework/1/CalcBMI.cs (hunk 1 above)
Comment (Laimonukas) [reply Homework/1/CalcBMI.cs]: Added age, weight and height functions.

**Review by Almantask — APPROVED** [review]

**Comment by Almantask** [comment]
looks good

**Pull request closed without merging**